filament/src/Renderer.cpp

```diff
@@ -588,7 +588,23 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
 
     // TAA for color pass
     if (taaOptions.enabled) {
-        input = ppm.taa(fg, input, view.getFrameHistory(), taaOptions, colorGradingConfig);
+        input = ppm.taa(fg, input, view.getFrameHistory(), getColorHistory(fg, view), taaOptions,
+                colorGradingConfig);
+    }
+
+    struct SaveColorHistoryData {
+        FrameGraphId<FrameGraphTexture> color;
+    };
+    if (taaOptions.enabled) {
+        fg.addPass<SaveColorHistoryData>("Save color history", [&](FrameGraph::Builder& builder,
+                auto& data) {
+            builder.sideEffect();
```

Comment:
explain that we need to use a `sideEffect` here (i.e.: pass won't be culled) because the "output" of this pass is going to be used during the next frame as an "import".